Sources/RawRepresentable+Arbitrary.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  RawRepresentable+Arbitrary.swift
+//  SwiftCheck
+//
+//  Created by Brian Gerstle on 5/4/16.
+//  Copyright © 2016 Typelift. All rights reserved.
+//
+
+extension RawRepresentable where RawValue: Arbitrary {
```

Comment:
Yeah

On Thu, May 5, 2016 at 11:31 AM Robert Widmann <notifications@github.com>
wrote:

> In Sources/RawRepresentable+Arbitrary.swift
> <https://github.com/typelift/SwiftCheck/pull/160#discussion_r62204186>:
>
> > @@ -0,0 +1,14 @@
> > +//
> > +//  RawRepresentable+Arbitrary.swift
> > +//  SwiftCheck
> > +//
> > +//  Created by Brian Gerstle on 5/4/16.
> > +//  Copyright © 2016 Typelift. All rights reserved.
> > +//
> > +
> > +extension RawRepresentable where RawValue: Arbitrary {
>
> Can this be moved among the other Arbitrary instances?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/typelift/SwiftCheck/pull/160/files/740068cfb37737c00e528527cac3a517299b0fb1#r62204186>
>